ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/_layout/runs/$runIndex/index.tsx

```diff
@@ -46,6 +49,14 @@ const RunComponent = () => {
         repositoryId: Number.parseInt(params.repoId),
         ortRunIndex: Number.parseInt(params.runIndex),
       }),
+    refetchInterval: (run) => {
+      if (
+        run.state.data?.status === 'FINISHED' ||
+        run.state.data?.status === 'FAILED'
+      )
+        return false;
```

Comment:
Shouldn't this have `{}`?

Reply:
Apparently works without. Are they auto-formatted away if you add them?

Reply:
I've never faced it before, only in one liners like `if (condition) return true` but not separate lines and indicated by indentation. IMO it's clearer with braces, but as the formatter doesn't care, feel free to do as you see best.